docs/tools/terraform/reference/cookbook/kafka-debezium-postgres-source.rst

```diff
@@ -0,0 +1,197 @@
+Apache Kafka® with Debezium source connector for PostgreSQL®
+==========================================================
+
+The `Aiven Terraform Provider <https://registry.terraform.io/providers/aiven/aiven/latest/docs>`_ is a great choice for provisioning an Aiven for Apache Kafka® cluster with Kafka Connect enabled and the `Debezium source connector for PostgreSQL® <https://developer.aiven.io/docs/products/kafka/kafka-connect/howto/debezium-source-connector-pg.html>`_ configured.
+
+Let's check out the following diagram to understand the setup.
+
+.. mermaid::
+
+   flowchart LR
+      id1[(PostgreSQL Database)]
+      subgraph Kafka Connect
+      Debezium-Source-Connector-PG
+      end
+      subgraph Apache Kafka
+      Topic
+      end
+      id1 --->|wal2json plugin| Debezium-Source-Connector-PG -->|publishes changes| Topic
+
+Describe the setup
+------------------
+
+This terraform recipe will provision one Aiven PostgreSQL database service, one Aiven for Apache Kafka service, a separate Aiven for Apache Kafka Connect 
+service with a Debezium source connector for PostgreSQL enabled and configured to connect to the PostgreSQL database and capture any change in tables.
+As soon as any of the monitored tables is inserted or updated with new data, the Debezium connector will capture the data change and convert table data into
+json payload and produce messages to the relevant Kafka topic. 
+Some of these services are created in one cloud provider and some in another cloud provider, to demonstrate how easy it is with Aiven to integrate services across 
+multiple cloud vendors.
+
+.. Warning::
+
+    Creating a `dedicated Aiven for Apache Kafka Connect service <https://developer.aiven.io/docs/products/kafka/kafka-connect/getting-started.html#apache-kafka-connect-dedicated-cluster>`_ is Aiven's suggested option. Having Kafka Connect running on the same nodes as Apache Kafka increases the load on the nodes possibly making the cluster more unstable. 
+    
+    For a better separation of concerns, a dedicated Aiven for Apache Kafka Connect cluster is therefore suggested.
+
+
+.. Tip::
+
+    Be sure to check out the :doc:`getting started guide <../../get-started>` to learn about the common files required to execute the following recipe.
+    For example, you'll need to declare the variables for ``project``, ``api_token``.
+    These global variables that allow terraform to access the Aiven console and instruct deployments, are normally configured in a separate file called ``secret.tfvars`` and called 
+    in the terraform command line using the option ``--var-file=<path/to/tfvars file>``.
+
+The content of the ``secret.tfvars`` file, looks like this:
+
+.. code::
+
+  project = "my-aiven-project"
+  aiven_api_token = "<AIVEN_TOKEN>"
+  prefix = "anytext"
+
+The ``service.tf`` file for the provisioning af these 3 services and integrations is this:
+
+.. code::
```

Comment:
please mark as terraform here for highlighting

Reply:
does this work? I think is not supported

Reply:
i kinda randomly found that pygments supports terraform: https://pygments.org/docs/lexers/, we already use it i think for other recipes too